config/resource-manager.hcl

```
@@ -33,7 +33,7 @@ service "applicationinsights" {
 }
 service "appplatform" {
   name      = "AppPlatform"
-  available = ["2022-09-01-preview", "2022-12-01", "2023-05-01-preview", "2023-07-01-preview"]
+  available = ["2022-09-01-preview", "2022-12-01", "2023-05-01-preview", "2023-07-01-preview", "2023-09-01-preview"]
```

Since we're no longer using the older versions, can we remove these too

```suggestion
  available = ["2023-07-01-preview", "2023-09-01-preview"]
```